### ISSUE TITLE
Writing a LazyBufferedTIFF and can't save the added tags

### ISSUE THREAD

**Issue by dimer116** [tlnagy/TiffImages.jl#105]
Hi and thanks for the help in advance. I am trying to write additional tags to lazybufferedtiffs but I am not managing to get the tags saved. I am also writing some tiffs that are >4GB so I need bigtiff=true. Is there a way to do this? Here is some example code:

```
using TiffImages
using FixedPointNumbers
using Colors

random_data = rand(Gray{N0f16}, 100, 100)

lazy_tiff = empty(LazyBufferedTIFF, Gray{N0f16},"test1.tiff"; bigtiff=true) 
push!(lazy_tiff, random_data)
lazy_tiff.ifds[1][TiffImages.IMAGEDESCRIPTION] = "Hello world"

TiffImages.save("test2.tiff", lazy_tiff) 

normal_tiff = TiffImages.load("test2.tiff")
normal_tiff.ifds[1][TiffImages.IMAGEDESCRIPTION] = "Hello world"
TiffImages.save("test3.tiff", normal_tiff) 

test1_tiff = TiffImages.load("test1.tiff") # Doesn't have tag
test2_tiff = TiffImages.load("test2.tiff") # Doesn't have tag
test3_tiff = TiffImages.load("test3.tiff") # Has tag
```

**Comment by tlnagy** [comment]
The current implementation of [`push!`](https://github.com/tlnagy/TiffImages.jl/blob/b39ec2e87b2d0dfdb62d3c591950da35d9431542/src/types/lazy.jl#L141-L172) isn't flexible enough to support this. You tweak that function slightly to accept an IFD as a parameter as well instead of making a new one automatically [here](https://github.com/tlnagy/TiffImages.jl/blob/b39ec2e87b2d0dfdb62d3c591950da35d9431542/src/types/lazy.jl#L156). I would be happy to review a PR if you're interested in sharing your solution.

**Comment by tlnagy** [comment]
This limitation is fixed in #109 and will be in the next version of TiffImages `v0.6.6`. 

All you need to do is construct a 2D `DenseTaggedImage` with your desired tags added to its IFD and then `push!` it to the `LazyBufferedTIFF`:

```julia
julia> slice = TiffImages.DenseTaggedImage(Gray.(zeros(UInt8, 10, 10)));

julia> first(slice.ifds)[TiffImages.IMAGEDESCRIPTION] = "Custom info";

julia> temp_file = joinpath(mktempdir(), "tmp.tif");

julia> lazy_tiff = empty(LazyBufferedTIFF, Gray{N0f8}, temp_file; bigtiff = true);

julia> push!(lazy_tiff, slice);

julia> close(lazy_tiff) # done writing, close open stream

julia> first(TiffImages.load(temp_file).ifds)[TiffImages.IMAGEDESCRIPTION] # read from disk
Tag(IMAGEDESCRIPTION, "Custom info")
```
